src/Sarif.Viewer.VisualStudio.Core/Tags/SarifLocationErrorTag.cs

```diff
@@ -42,6 +72,101 @@ public SarifLocationErrorTag(IPersistentSpan persistentSpan, int runIndex, int r
         /// <remarks>
         /// This may be null.
         /// </remarks>
-        public object ToolTipContent { get; }
+        private readonly List<(string strContent, TextRenderType renderType)> content;
+
+        public object ToolTipContent
+        {
+            get
+            {
+                ThreadHelper.ThrowIfNotOnUIThread();
+                foreach ((string strContent, TextRenderType renderType) item in content)
+                {
+                    if (item.renderType == TextRenderType.Markdown)
+                    {
+                        try
+                        {
+                            MarkdownViewer viewer = new MarkdownViewer();
+                            viewer.Markdown = item.strContent;
+                            ParseBlocks(viewer.Document.Blocks);
+
+                            viewer.Margin = new Thickness(-15, -15, 0, 0); // There is a small amount of padding that MarkdownViewer comes with that makes it awkward when a textfield is put alongside it.
+                            return viewer;
+                        }
+                        catch (Exception)
```

Comment:
>Exception [](http://example.com/codeflow?start=31&length=9)

If possible, only catch the exceptions you know to expect. If others are thrown, we want to see it happen during testing. (I know this is done hundreds of times in this codebase but it's a bad practice)